e2etests/test/device_provision.js

```diff
@@ -72,13 +76,58 @@ var device_provision = function (hubConnectionString, done) {
     });
   }
 
+  function createCACertDevice(deviceId, done) {
+    var certOptions = {
```

Comment:
Removed

---
In reply to: [128300745](https://github.com/Azure/azure-iot-sdk-node/pull/61#discussion_r128300745) [](ancestors = 128300745)